R/calc.R

```diff
@@ -2,7 +2,7 @@
 #' @name calc
 #' @param ph.data data.table or tbl_svy. Dataset.
 #' @param what character vector. Variable to calculate metrics for.
-#' @param ... expressions to be passed to \code{\link{filter}} (or equivalent)
+#' @param where subsetting expression
```

Comment:
here you specify a where argument, but code doesn't seem to require it it. Could we force it to use a where (knowing it will break backward compatibility)?

Reply:
What do you mean forcing a where? What if no subsetting is required?

Reply:
My apologies. If no subsetting is required, then there would be no need for where. I meant forcing the user to explicitly type something like `where = 'age %in% 10:20'` rather than `age %in% 10:20`